bot_modules/util.js

```diff
@@ -22,20 +22,23 @@ var self = module.exports = {
 		if ((months == 0 && days < 0) || (months < 0)) years--;
 		if (months < 0) months += 12;
 		if (days < 0) {
-			months--;
-    	var days_in_future_month  = new Date(future_date.getFullYear(),  future_date.getMonth()  + 1, 0).getDate();
-    	var days_in_present_month = new Date(current_date.getFullYear(), current_date.getMonth() + 1, 0).getDate();
-    	var days_left_in_present_month = days_in_present_month - current_date.getDate();
-    	days = future_date.getDate() + days_left_in_present_month;
- 		}
-		var formatted_ymd = days + " days";
-		if (years > 0) {
-			formatted_ymd = years + " years " + months + " months " + formatted_ymd;
+			months--
+			var days_in_future_month  = new Date(
+				future_date.getFullYear(),
+				future_date.getMonth()  + 1,
+				0).getDate();
+			var days_in_present_month = new Date(
+				current_date.getFullYear(),
+				current_date.getMonth() + 1,
+				0).getDate();
+			var days_left_in_present_month = days_in_present_month - current_date.getDate();
+		days = future_date.getDate() + days_left_in_present_month;
 		}
-		else if (months > 0) {
-			formatted_ymd = months + " months " + formatted_ymd;
-		}
-		return formatted_ymd;
+		return years > 0 ? 
+			`${years} years ${months} months ${days} days` : 
+			months > 0 ?
+				`${months} months ${days} days` :
+				`${days} days`;
```

Comment:
double ternaries are ambiguous. always wrap those suckers up.

this guy:

```
a ? b : c ? d : e
```

can be interpreted as

```
a ? b : (c ? d : e)
```

or

```
(a ? b : c) ? d : e
```

it might be in the spec but you can save future coders the time of looking it up by wrappin it up, 🅱️

Reply:
addendum, i don't like long ternaries personally. i would pull one of them out into a variable myself

Reply:
i was thinkin this too

Reply:
squashed into commit